clang/test/dpct/cmake_migation/case_004/case_004_target_link_libraries.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: rm -rf %T && mkdir -p %T
+// RUN: cd %T
+// RUN: cp %S/input.cmake ./input.cmake
+// RUN: dpct -in-root ./ -out-root out  ./input.cmake --rule-file=%S/../../../../tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml --migrate-cmake-script-only --cuda-include-path="%cuda-path/include"
```

Comment:
--rule-file=%S/../../../../tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml and --cuda-include-path="%cuda-path/include" could be removed.

Reply:
Done.